apps/easypid/src/agent/initialize.ts

```diff
@@ -3,7 +3,7 @@ import { trustedX509Certificates } from '@easypid/constants'
 import { WalletServiceProviderClient } from '@easypid/crypto/WalletServiceProviderClient'
 import { isFunkeWallet } from '@easypid/hooks/useFeatureFlag'
 import { initializeEasyPIDAgent } from '@package/agent'
-import { initializeParadymAgent, isEasyPIDAgent } from '@package/agent/src/agent'
+import { initializeParadymAgent, isEasyPIDAgent } from '@package/agent/agent'
```

Comment:
Maybe a small check that everything is imported from root?  It works both ways, but `@package/agent` feels a bit better to me.